### PR TITLE
[Docs] Fix WSL1 hide/show button UX issue

### LINKED ISSUE #2026: [Docs] Show/hide button UX issues
**Current State:**
A show/hide button is being used in some of the Meshery docs. It has two usability issues:

1. It is not obvious that users can click on the show/hide buttons like [this one for WSL](https://meshery.layer5.io/docs/installation/platforms/windows#choosing-your-wsl-version).
2. The button acts differently depending upon whether the user fully depresses the button or not.
 
Dec-03-2020 01-08-09

**Desired State:**

1. Change the cursor to a pointer on hover.
2. Make the click action consistent in its behavior.

---
**Contributor Resources**
- [Meshery documentation site](https://meshery.layer5.io/docs/)
- [Meshery documentation source](https://github.com/layer5io/meshery/tree/master/docs)
- [Instructions for contributing to documentation](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)

### PR DESCRIPTION
**Pull request by Shulammite-Aso**
Signed-off-by: Shulammite-Aso <asoamarachi@gmail.com>

**Description**

This PR fixes #2026 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Shulammite-Aso** [comment]
Hi :raising_hand:  @leecalcote Please review.

**Comment by leecalcote** [comment]
Oh! very good. 👀

File: docs/pages/es/installation/windows.md [hunk 1]
```diff
@@ -83,7 +83,7 @@ wsl --set-default-version 2
 <b>Advertencia:</b>
 Se recomienda actualizar a <a href="#wsl2">WSL2</a> ya que WSL1 no soporta la aplicación Docker Desktop para Windows. En su lugar, solo soporta la versión obsoleta, [Docker Toolbox](https://docs.docker.com/toolbox/toolbox_install_windows/).
 
-Si aun desea continuar, siga las instrucciones para <button onclick="HideToggleFunction()"><b>WSL1</b></button>
+Si aun desea continuar, siga las instrucciones para <button class="WSL1-button" onclick="HideToggleFunction()"><b>WSL1</b></button>
```
Comment (leecalcote) [inline docs/pages/es/installation/windows.md:86]: Good work, @Shulammite-Aso. You accounted for the Spanish version, too. 😄

Comment (Shulammite-Aso) [reply docs/pages/es/installation/windows.md:86]: Yeah @leecalcote :laughing:  These were the two places i saw the button in use.

**Comment by chandrashritii** [comment]
Looks great. The button is much more obvious now. Good work! @Shulammite-Aso

**Review by chandrashritii — APPROVED** [review]

**Comment by leecalcote** [comment]
@chandrashritii is this button to be broadly used? If so, let’s consider a class name change.

**Comment by chandrashritii** [comment]
Yes @leecalcote 
@Shulammite-Aso Could you change the class name for the button to something more generic than WSL1-button?

**Comment by Shulammite-Aso** [comment]
> Yes @leecalcote
> @Shulammite-Aso Could you change the class name for the button to something more generic than WSL1-button?

Okay @chandrashritii , will _toggle-button_ work? if not exactly, could you give an example?

**Comment by leecalcote** [comment]
@Shulammite-Aso yes, sounds real good. This kind of functionality is useful on a number of other pages.

**Comment by Shulammite-Aso** [comment]
@leecalcote @chandrashritii done.

**Review by chandrashritii — APPROVED** [review]

**Comment by leecalcote** [comment]
Yay! @Shulammite-Aso 🎈

**Pull request merged**